CMakeLists.txt

```diff
@@ -600,6 +600,18 @@ else()
   message(STATUS "JNI library is disabled")
 endif()
 
+# Installation and packaging for Linux
+include(GNUInstallDirs)
+install(TARGETS ${ROCKSDB_STATIC_LIB} COMPONENT devel ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
+install(TARGETS ${ROCKSDB_SHARED_LIB} COMPONENT runtime DESTINATION ${CMAKE_INSTALL_LIBDIR})
```

Comment:
On Windows shared library produce two files DLL and LIB. Can be controlled by `ARCHIVE DESTINATION` and `RUNTIME DESTINATION`. Usually I'm using [this code](http://cgold.readthedocs.io/en/latest/tutorials/install/layout.html):
```cmake
install(
    TARGETS <project>_<target>_1 <project>_<target>_2
    EXPORT <project>Targets
    LIBRARY DESTINATION "lib"
    ARCHIVE DESTINATION "lib"
    RUNTIME DESTINATION "bin"
    INCLUDES DESTINATION "include"
)
```

Works for most cases if the target is not something exotic like OSX bundle/framework.